runtime/src/main/cpp/Arrays.cpp

```diff
@@ -379,6 +404,7 @@ KFloat Kotlin_FloatArray_get(KConstRef thiz, KInt index) {
   if (static_cast<uint32_t>(index) >= array->count_) {
     ThrowArrayIndexOutOfBoundsException();
   }
+  mutabilityCheck(thiz);
```

Comment:
Mutability check in array `get`?

Reply:
Mutability shall only be tested in setter, isn't it?

Reply:
Exactly, and this function is named `Kotlin_FloatArray_get`. Is it ok?

Reply:
OK